ethereum/env/.env.seievm

```diff
@@ -0,0 +1,13 @@
+# Sei EVM mainnet read only env. Use to deploy the core contract with forge.
+
+# ethereum$ ln -s env/.env.seievm .env
+
+RPC_URL=""
```

Comment:
I think the RPC for arctic-1 is `https://evm-rpc-arctic-1.sei-apis.com/` [[source](https://v2.docs.sei.io/setting-up-a-wallet#switching-networks)]